src/main/java/mate/academy/library/model/ShoppingCart.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.library.model;
+
+import jakarta.persistence.Entity;
+import jakarta.persistence.FetchType;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import jakarta.persistence.JoinColumn;
+import jakarta.persistence.OneToMany;
+import jakarta.persistence.OneToOne;
+import jakarta.persistence.Table;
+import lombok.Getter;
+import lombok.RequiredArgsConstructor;
+import lombok.Setter;
+import lombok.ToString;
+import java.util.Set;
+
+@Entity
+@Getter
+@Setter
+@RequiredArgsConstructor
+@Table(name = "shopping_carts")
+public class ShoppingCart {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+    @OneToOne
+    @JoinColumn(name = "user_id", nullable = false)
+    private User user;
+    @ToString.Exclude
+    @OneToMany(mappedBy = "shoppingCart", fetch = FetchType.LAZY)

```

Comment:
```suggestion
    @OneToMany(mappedBy = "shoppingCart", cascade = CascadeType.ALL, orphanRemoval = true)
```

OneToMany has fetchtype lazy by default

Reply:
Not fixed